src/encoded/static/components/gene.js

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as globals from './globals';
+import { Breadcrumbs } from './navigation';
+import { DbxrefList, dbxrefHref } from './dbxref';
+import { PickerActions } from './search';
+import { auditDecor } from './audit';
+
+
+const Gene = (props) => {
+    const context = props.context;
+    const itemClass = globals.itemClass(context, 'view-detail key-value');
+
+    // Set up breadcrumbs
+    const crumbs = [
+        { id: 'Genes' },
+        {
+            id: <i>{context.organism.scientific_name}</i>,
+            query: `organism.scientific_name=${context.organism.scientific_name}`,
+            tip: `${context.organism.scientific_name}`,
+        },
+    ];
+
+    return (
+        <div className={globals.itemClass(context, 'view-item')}>
+            <header className="row">
+                <div className="col-sm-12">
+                    <Breadcrumbs root="/search/?type=gene" crumbs={crumbs} />
+                    <h2>{context.symbol} (<em>{context.organism.scientific_name}</em>)</h2>
+                </div>
+            </header>
+
+            <div className="panel">
+                <dl className={itemClass}>
+                    {/* TODO link to NCBI Entrez page? */}
+                    <div data-test="gendid">
+                        <dt>Entrez GeneID</dt>
+                        <dd>
+                            <a href={dbxrefHref('GeneID', context.geneid)}>{context.geneid}</a>
```

Comment:
All looks good! Except I’m not sure why you need `dbxrefHref` here instead of `<DbxrefList>` which would put in the links for you? `dbxrefHref` is only for very unusual cases (Targets needed something odd, and I don’t remember why). If the issue is that `<DbxrefList>` needs an array, you can pass `dbxrefs={[context.geneid]}`, or maybe you could make `<DbxrefUrl>` external and use that instead. We just never needed that before, but no problem having it external now.

Reply:
I don't have a good reason. I want to make the text in display just the ID while I thought I'll need to pass "GeneID:{context.geneid}" to DbxrefList which will make the display have a "GeneID" prefix. Is that right?

Reply:
That’s right.